adding-to-and-updating-the-cart.md

```diff
@@ -118,28 +119,28 @@ You can add multiple purchasables to the cart in an update cart form. You supply
     {{ csrfInput() }}
 
     {% for variant in product.variants %}
-        <input type="hidden" name="purchasables[{{loop.index}}][id]" value="{{ variant.id }}">
-        <input type="hidden" name="purchasables[{{loop.index}}][qty]" value="1">
-        <input type="hidden" name="purchasables[{{loop.index}}][note]" value="1">
+        <input type="hidden" name="purchasables[{{ loop.index }}][id]" value="{{ variant.id }}">
+        <input type="hidden" name="purchasables[{{ loop.index }}][qty]" value="1">
+        <input type="hidden" name="purchasables[{{ loop.index }}][note]" value="1">
     {% endfor %}
 
     <input type="submit" value="Add all variants to cart">
 </form>
 ```
 
-While using multi-add the same rules apply for updating a quantity vs adding to cart, based on the uniquessness of the options `signature` and `purchasableId`.
+While using multi-add the same rules apply for updating a quantity vs. adding to cart, based on the uniquessness of the options `signature` and `purchasableId`.
 
-As shown in the example above,  a unique index key is required to group the purchasable ID to its related `notes` and `options` and `qty` param. Using `{{loop.index}}` is an easy way to do this.
+As shown in the example above, a unique index key is required to group the purchasable ID to its related `notes` and `options` and `qty` param. Using `{{ loop.index }}` is an easy way to do this.
 
 ## Updating line items
 
 Once the purchasable has been added to the cart, your customer may want to update the `qty` or `note`, they can do this by updating a line item.
 
-Line items can have their `qty`, `note`, and `options`updated. They can also be removed.
+Line items can have their `qty`, `note`, and `options` updated or removed.
 
-To update a line item, submit a form array param with the name of `lineItems`, with the ID of the array key being the line item ID.
+To update a line item, submit a `lineItems` form array, where each item’s array key is the line item ID.
 
-Example:
+Example using `LINE_ITEM_ID` as a placeholder for the ID of the item you’d like to edit:
```

Comment:
Lets get rid of the `LINE_ITEM_ID` example, and just add a loop over light items in the example and use `{{ item.id }}`